public/viewtopic.php

```diff
@@ -380,7 +380,7 @@ class="fullwidth"
             echo "</tbody></table></div>";
         } else {
             echo "</tbody></table></div>";
-            RenderLoginComponent($user, $points, $errorCode, true);
+            echo "<br/>You must log in before you can join this conversation.<br/>";
```

Comment:
This could have passed `$userDetails['RAPoints']` as a replacement for `$points`, but having a second set of login fields on the page felt awkward.

Reply:
Thanks, that was bothering me for some time now